packages/bierzo-wallet/src/routes/balance/index.e2e.spec.ts

```diff
@@ -76,6 +76,7 @@ withChainsDescribe("E2E > Balance route", () => {
     const buttons = await page.$$("button");
     await buttons[2].click();
 
+    await sleep(1500);
```

Comment:
I'm not a big fan of adding sleeps that are not needed most of the time, as this will increase the test runtime more and more. Instead, we should try to use `whenTrue` if possible, that will resolve very quickly if the test condition is met.

In this case we can build something like

```ts
await whenTrue(async () => (await getUsernameE2E(await page.$$("h5"))).matches(/\*iov$/), 5000);
```

which we can also wrap in a helper similar to `waitForAllBalances`